tuf-spec.md

```diff
@@ -1116,16 +1116,29 @@ repo](https://github.com/theupdateframework/specification/issues).
   cycle, report the potential freeze attack.  On the next update cycle, begin
   at step 0 and version N of the root metadata file.
 
-  * **1.9**. **If the timestamp and / or snapshot keys have been rotated, then
-  delete the trusted timestamp and snapshot metadata files.** This is done in
-  order to recover from fast-forward attacks after the repository has been
-  compromised and recovered. A _fast-forward attack_ happens when attackers
-  arbitrarily increase the version numbers of: (1) the timestamp metadata, (2)
-  the snapshot metadata, and / or (3) the targets, or a delegated targets,
-  metadata file in the snapshot metadata. Please see [the Mercury
+  * **1.9**. **Fast-forward attack recovery** A _fast-forward attack_ happens
+  when attackers arbitrarily increase the version numbers in any of the
+  timestamp, snapshot, targets, or delegated targets metadata. To recover from
```

Comment:
```suggestion
  timestamp, snapshot, targets, or delegated targets metadata. The attacker goal
  is to cause clients to refuse to update the metadata later because the attacker's
  listed metadata version number (possibly MAX_INT) is greater than the new valid 
  version.  To recover from
```

Reply:
Would recommend something like this...

Reply:
Good addition. Will add...